crates/oxc_ast/src/ast/js.rs

```diff
@@ -2116,6 +2150,7 @@ pub struct AccessorProperty<'a> {
     /// Decorators applied to the accessor property.
     ///
     /// See [`Decorator`] for more information.
+    #[ts]
```

Comment:
I think accessor properties themselves are ts-only, but I'm not certain

Reply:
Yes, I believe that's correct.

Reply:
No, it's a JavaScript feature
https://github.com/tc39/proposal-decorators?tab=readme-ov-file#new-class-elements

Reply:
Well that depends on whether we consider decorators to be JS or TS. As discussed in other comments here, I think we decided to treat them as TS for now.

I don't think it matters so much either way. It'll be easy to change later on - we just remove the `#[ts]` attrs from decorator-related fields, and the code will update. Such is the joy of codegen!